src/client/components/general/ProductCard/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Button, Tag} from '../../index'
+import './productCard.scss'
+
+function ProductCard({title, description, price, img, tag, index }) {
+  console.log(index)
+
+  return (
+    <article className='product' key={index}>
+        { tag ? <Tag text={tag.text} theme={tag.theme} /> : ''}
+        <img src={img} alt='shampoo bottle' className='product-img'></img>
```

Comment:
check all alt's throughout the all components in the applications - if it's a component which image is also passed as a prop